tasks/libs/pipeline/github_slack_review_map.yaml

```diff
@@ -0,0 +1,48 @@
+# This file contains a mapping of DataDog Github teams to slack channels for reviews.
+# The DEFAULT_SLACK_CHANNEL value is interpreted as '#agent-devx-ops'.
+# Note that the values must be quoted if they contain a '#' symbol, because
+# YAML interprets that as the beginning of a comment. Keys must also be quoted
+# because the '@' symbol is reserved in YAML.
+'@datadog/agent-platform': DEFAULT_SLACK_CHANNEL
+'@datadog/documentation': '#documentation'
+'@datadog/container-integrations': '#container-integrations'
+'@datadog/container-platform': '#container-platform'
+'@datadog/container-ecosystems': '#container-ecosystems'
+'@datadog/platform-integrations': '#platform-integrations'
+'@datadog/agent-security': '#security-and-compliance-agent'
+'@datadog/agent-apm': '#apm-agent'
+'@datadog/network-device-monitoring': '#network-device-monitoring'
+'@datadog/ndm-core': '#ndm-core'
+'@datadog/ndm-integrations': '#ndm-integrations'
+'@datadog/container-intake': '#process-agent-ops'
+'@datadog/agent-metrics-logs': '#agent-metrics-logs'
+'@datadog/agent-processing-and-routing': '#agent-processing-and-routing'
+'@datadog/agent-shared-components': '#agent-shared-components'
+'@datadog/container-app': '#container-app'
+'@datadog/metrics-aggregation': '#metrics-aggregation'
+'@datadog/serverless': '#serverless-agent'
+'@datadog/remote-config': '#remote-config-monitoring'
+'@datadog/fleet': '#fleet-automation-monitoring'
+'@datadog/agent-all': '#datadog-agent-pipelines'
+'@datadog/ebpf-platform': '#ebpf-platform'
+'@datadog/networks': '#network-performance-monitoring'
+'@datadog/universal-service-monitoring': '#universal-service-monitoring'
+'@datadog/windows-agent': '#windows-agent-reviews'
+'@datadog/windows-kernel-integrations': '#windows-kernel-integrations-reviews'
+'@datadog/opentelemetry': '#opentelemetry'
+'@datadog/agent-e2e-testing': '#agent-e2e-framework-devs'
+'@datadog/software-integrity-and-trust': '#sit'
+'@datadog/single-machine-performance': '#single-machine-performance'
+'@datadog/agent-integrations': '#agent-integrations'
+'@datadog/debugger': '#debugger'
+'@datadog/database-monitoring': '#database-monitoring'
+'@datadog/agent-cspm': '#k9-cspm'
+'@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
+'@datadog/apm-trace-storage': '#apm-trace-storage'
+'@datadog/asm-go': '#k9-library-go'
+'@datadog/agent-delivery': '#agent-delivery-reviews'
+'@datadog/agent-devx-infra': '#agent-devx-infra-reviews'
+'@datadog/agent-devx-loops': '#agent-devx-loops-reviews'
+'@datadog/apm-onboarding': '#apm-onboarding'
+'@datadog/apm-ecosystems-performance': '#apm-benchmarking-platform'
+'@DataDog/container-ecosystems': '#container-ecosystems'
```

Comment:
```suggestion
'@datadog/container-ecosystems': '#container-ecosystems'
```

Reply:
This entry was duplicated, for some reason